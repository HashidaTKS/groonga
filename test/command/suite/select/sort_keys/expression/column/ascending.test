table_create Values TABLE_NO_KEY
column_create Values number COLUMN_VECTOR Int32

load --table Values
[
{"number": 128},
{"number": 255},
{"number": -254}
]

select Values --sort_keys '1 + 1, +number' --output_columns number
