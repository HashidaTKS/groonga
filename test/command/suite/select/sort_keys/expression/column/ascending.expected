table_create Values TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Values number COLUMN_VECTOR Int32
[[0,0.0,0.0],true]
load --table Values
[
{"number": 128},
{"number": 255},
{"number": -254}
]
[[0,0.0,0.0],3]
select Values --sort_keys '1 + 1, +number' --output_columns number
[[0,0.0,0.0],[[[3],[["number","Int32"]],[[-254]],[[128]],[[255]]]]]
